### ISSUE TITLE
Texture transparency is broken

### ISSUE THREAD

**Issue by dmitrykolesnikovich** [husker-dev/openglfx#27]
So, in my game engine I heavily rely on JavaFX and I consider your OpenGLFX library as the only solution for the [Windows 10 bug](https://bugs.openjdk.org/browse/JDK-8222317) that makes JavaFX completely unsusable. I'm commited to reach production quality of your library hopefully with your help :) Here is another bug report - you can reproduce it by trying [my pull request](https://github.com/husker-dev/openglfx/pull/26) (specifically [this commit](https://github.com/husker-dev/openglfx/pull/26/commits/4abca1f5ff42504fab0bdda76357418f0f9eb939))

Basically what's intended to be transparent is not.

![image](https://user-images.githubusercontent.com/6131040/216362971-6687c6e9-0fe8-4dfa-9703-7b324732fb8b.png)

**Comment by dmitrykolesnikovich** [comment]
not a bug because https://github.com/dmitrykolesnikovich/openglfx/commit/bc6e4a951e7cc9978e8c1c95fe35d0f79c0977f5
